samples/FunctionApp/Program.cs

```diff
@@ -2,52 +2,39 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Linq;
-using System.Threading.Tasks;
+using FunctionApp;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 
-namespace FunctionApp
+//<docsnippet_startup>
+//<docsnippet_configure_defaults>
+FunctionsApplicationBuilder funcBuilder = FunctionsApplication.CreateBuilder(args);
+//</docsnippet_configure_defaults>
+
+//<docsnippet_dependency_injection>
+funcBuilder.Services.AddApplicationInsightsTelemetryWorkerService();
+funcBuilder.Services.ConfigureFunctionsApplicationInsights();
+funcBuilder.Services.AddSingleton<IHttpResponderService, DefaultHttpResponderService>();
+funcBuilder.Services.Configure<LoggerFilterOptions>(options =>
 {
-    class Program
+    // The Application Insights SDK adds a default logging filter that instructs ILogger to capture only Warning and more severe logs. Application Insights requires an explicit override.
+    // Log levels can also be configured using appsettings.json. For more information, see https://learn.microsoft.com/en-us/azure/azure-monitor/app/worker-service#ilogger-logs
+    LoggerFilterRule toRemove = options.Rules.FirstOrDefault(rule => rule.ProviderName
+        == "Microsoft.Extensions.Logging.ApplicationInsights.ApplicationInsightsLoggerProvider");
+    FunctionsApplicationBuilder builder = FunctionsApplication.CreateBuilder(args);
```

Comment:
Why do we initialize `FunctionsApplicationBuilder` here?

Reply:
Whoops. That was an accident in a last-second refactor. I'll remove it. 

In fact, I think we're going to add a new "FunctionApplicationBuilder" sample just so we don't swap out these existing ones and confuse anyone that may be linking to them. And some of the docs are generated from these, so I shouldn't change them without a bigger review.